<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>5</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionix - Dashboard</string>
  </property>
  <property name="styleSheet">
<<<<<<< Updated upstream
   <string notr="true">QFrame {
	border: none;
}

QLineEdit {
	padding: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
=======
   <string notr="true">QMainWindow {
	background-image: url(:/background/background_principal.png);
}

QLabel {
	color: black
}

QGroupBox {
	color: black;
	background-color: rgba(245, 245, 245, 0.8);
	border-radius: 5px;
	padding: 10px
}

QScrollArea {
	background-color: rgba(245, 245, 245, 0.8);
}

#scrollAreaWidgetContents {
	background-color: rgba(245, 245, 245, 0.8);
}

#title {
	color: #400b97
}

QPushButton {
	background-color: #400b97;
	color: white;
	border: none;
	border-radius: 10px
}

QPushButton:hover,
QPushButton:clicked {
	background-color: #400baa;
}

QScrollArea {
	border-radius: 12px;
}

QCombotBox {
	color: black;
	background-color: #400baa
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
>>>>>>> Stashed changes
    <item>
     <widget class="QFrame" name="home">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Products List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Product Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product Stock</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Stock price ($)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Options</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="container">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
<<<<<<< Updated upstream
           <widget class="QLabel" name="label">
=======
           <widget class="QPushButton" name="settingsButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="title">
>>>>>>> Stashed changes
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Gestionix</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
<<<<<<< Updated upstream
=======
          <item>
           <widget class="QPushButton" name="logoutButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
>>>>>>> Stashed changes
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addButtons">
           <property name="text">
            <string>New products</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="viewOrder">
           <property name="text">
            <string>View order</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sortCombo">
           <item>
            <property name="text">
             <string>Sort stock</string>
            </property>
<<<<<<< Updated upstream
           </item>
           <item>
            <property name="text">
             <string>Price</string>
=======
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Add Products</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sortButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Sort Products</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="searchButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Search Products</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="content" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
>>>>>>> Stashed changes
            </property>
           </item>
           <item>
            <property name="text">
             <string>Name</string>
            </property>
<<<<<<< Updated upstream
           </item>
           <item>
            <property name="text">
             <string>Stock</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Date</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="searchLine">
         <property name="placeholderText">
          <string>Search products</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Order list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <column>
          <property name="text">
           <string>Customer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Options</string>
          </property>
         </column>
=======
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="infosContainer">
               <item>
                <widget class="QLabel" name="nameLabel">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="priceLabel">
                 <property name="text">
                  <string>Price</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="stockLabel">
                 <property name="text">
                  <string>Stock</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="dateLabel">
                 <property name="text">
                  <string>Date</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="actionLabel">
                 <property name="text">
                  <string>Action</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignTop</set>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>784</width>
                  <height>208</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="leftMargin">
                  <number>5</number>
                 </property>
                 <property name="topMargin">
                  <number>3</number>
                 </property>
                 <property name="rightMargin">
                  <number>5</number>
                 </property>
                 <property name="bottomMargin">
                  <number>3</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="containerItems" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>50</height>
                    </size>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <property name="leftMargin">
                     <number>5</number>
                    </property>
                    <property name="topMargin">
                     <number>3</number>
                    </property>
                    <property name="rightMargin">
                     <number>5</number>
                    </property>
                    <property name="bottomMargin">
                     <number>3</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Bread</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>2,3€</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>x20</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>03/12/2000 - 09:20</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="comboBox">
                      <item>
                       <property name="text">
                        <string>Supprimer</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Modifier</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
>>>>>>> Stashed changes
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="myAccounts"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionuserAccounts">
   <property name="icon">
    <iconset>
     <normaloff>../static/accounts.svg</normaloff>../static/accounts.svg</iconset>
   </property>
   <property name="text">
    <string>userAccounts</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="myAccounts">
   <property name="text">
    <string>My account</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
